### DISCUSSION TITLE
How to configure GoReleaser to do a release in GitHub Actions for a library repository?

### DISCUSSION THREAD

**Discussion by mgingios** [goreleaser/goreleaser#3243] [Q&A] [answered]
How to configure GoReleaser to do a release in GitHub Actions for a library repository -- no binary?  I know the it is possible using the CLI with a  configuration file:

builds:
- skip: true

How can this be done in a CICD release script, such as a GitHub Action?

**Comment by caarlos0** [comment, marked as answer]
the same way binary releases work... just create the goreleaser workflow and the goreleaser config file...

example: https://github.com/caarlos0/env/blob/60b5226da3e5e4ca3dcb795b8416f30c5cb2c5a9/.github/workflows/build.yml#L47-L61

**Reply by mgingios** [reply]
Hello and thank you for your help.  I have a GitHub Action GoReleaser workflow file.  You mentioned ..."and the goreleaser config file".  I thought that the config file was only used by the goreleaser cli.  If it is used in CICD, where does it need to be placed?  Guessing the .github/workflows folder...  Can you please advise?

**Reply by mgingios** [reply]
I looked at your env repository and see that the .goreleaser.yaml configuration file is used in CICD, too.  I updated mine and things are working.  Thank you for your help!

**Reply by caarlos0** [reply]
the action basically downloads and run the cli... its the same thing.
